packages/server/src/users/users.service.spec.ts

```diff
@@ -0,0 +1,21 @@
+//import { Test, TestingModule } from '@nestjs/testing';
+//import { UsersService } from './users.service';
+
+describe('UserService', () => {
+  // let usersService = UsersService;
+
+  // beforeEach(async () => {
```

Comment:
NIT: Shouldn't have commented-out code left unless it's explicitly going to be used later